### DISCUSSION TITLE
Feature to disable OP as a whole

### DISCUSSION THREAD

**Discussion by 1ByteBit** [PurpurMC/Purpur#1555] [__Dismissed__] [answered]
Or at least lock it down to prevent adding more people as OP

This feature would be useful for server security, as people granting themselves OP is the main thing malicious parties attempt to do if given the chance. It would essentially lower the risk from situations such as Vulcans (now defunct) GUI or malicious plugins.

**Comment by RhythmicSys** [comment, marked as answer]
Already exists in base minecraft, change the `op-permission-level` to 0 in server.properties
[Minecraft permission levels](https://minecraft.wiki/w/Permission_level)

**Reply by 1ByteBit** [reply]
> Already exists in base minecraft, change the `op-permission-level` to 0 in server.properties
> [Minecraft permission levels](https://minecraft.wiki/w/Permission_level)

I feel stupid now lmao
